exporters/prometheus/src/main/java/io/opentelemetry/exporter/prometheus/internal/PrometheusComponentProvider.java

```diff
@@ -41,6 +48,19 @@ public MetricReader create(StructuredConfigProperties config) {
     if (host != null) {
       prometheusBuilder.setHost(host);
     }
+    String defaultHistogramAggregation = config.getString("metrics.default.histogram.aggregation");
```

Comment:
`PrometheusComponentProvider` is represents the prometheus implementation of file configuration. The schema for the config properties read here is defined in [opentelemetry-configuration](https://github.com/open-telemetry/opentelemetry-configuration/blob/670901762dd5cce1eecee423b8660e69f71ef4be/schema/meter_provider.json#L95-L118). 

I'm supportive of adding a PR to `opentelemetry-configuration` to add this property to file configuration, but let's hold off on this code until the corresponding schema exists.

Reply:
I've raised a PR (https://github.com/open-telemetry/opentelemetry-configuration/pull/97) regarding this.

Reply:
I think you meant to link to https://github.com/open-telemetry/opentelemetry-configuration/pull/99

Thanks! I'll take a look!

Reply:
So there's a sequence of things that need to happen to unblock this:
1. Merge spec [PR](https://github.com/open-telemetry/opentelemetry-specification/pull/4124) to define new option for prometheus exporter, which unblocks:
2. Merge [PR](https://github.com/open-telemetry/opentelemetry-configuration/pull/99#pullrequestreview-2154044980) to define new `default_histogram_aggregation` option in `opentelemetry-configuration`
3. Cut a release of `opentelemetry-configuration`
4. Update `opentelemetry-java` new version of `opentelemetry-configuration`, and update `PrometheusComponentProvider` as you've done here.

Obviously that's a lot of steps. The fastest way to make this new config option usable is to wait on updating `PrometheusComponentProvider`, and rely on programmatic config for the time being (and maybe the env var if we accept it).

Reply:
I agree with you. We can probably add config to `PrometheusComponentProvider`, once `open telemetry-configuration` is released. For now, I've removed the config from there.